### PR TITLE
Get notebooks in examples/sklearn/ to work in Google colab (Part 2/3) [#380] 

### PR DESCRIPTION
**Pull request by dharmod**
Signed-off-by: Modhu Dhar <dhar.modhumita@gmail.com>

Issue:  ModuleNotFoundError: No module named 'aif360', added code to install aif360 at the start

### CODE REVIEW

**Comment by anupamamurthi** [comment]
Hi @dharmod 
can you refer to https://github.com/anupamamurthi/AIF360/blob/95952f5ff2ee3c903fd357e0f3a0a06297272582/examples/demo_lime.ipynb

Can you add the "open in colab" badge 

you will need a markdown that points to your notebook

    "[Open In Colab](https://colab.research.google.com/github/Trusted-AI/AIF360/blob/master/examples/demo_lime.ipynb)\n"

**Comment by dharmod** [comment]
> Hi @dharmod can you refer to https://github.com/anupamamurthi/AIF360/blob/95952f5ff2ee3c903fd357e0f3a0a06297272582/examples/demo_lime.ipynb
> 
> Can you add the "open in colab" badge
> 
> you will need a markdown that points to your notebook
> 
> ```
> [Open In Colab](https://colab.research.google.com/github/Trusted-AI/AIF360/blob/master/examples/demo_lime.ipynb)
> ```

Done

File: examples/sklearn/demo_learning_fair_representations.ipynb [hunk 1]
```diff
@@ -1,2025 +1,2140 @@
...
-       "      <td>0</td>\n",
-       "      <td>0</td>\n",
-       "    </tr>\n",
-       "  </tbody>\n",
-       "</table>\n",
-       "<p>28941 rows × 93 columns</p>\n",
-       "</div>"
+      "cell_type": "markdown",
+      "source": [
+        "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/Trusted-AI/AIF360/blob/master/examples/demo_lime.ipynb)"
```
Comment (hoffmansc) [inline examples/sklearn/demo_learning_fair_representations.ipynb:6]: this links to `examples/demo_lime.ipynb`. Can you change it to `examples/sklearn/demo_learning_fair_representations.ipynb`?

File: examples/sklearn/demo_mdss_bias_scan.ipynb [hunk 2]
```diff
@@ -1,805 +1,895 @@
...
-       "      <td>F</td>\n",
-       "      <td>4</td>\n",
-       "    </tr>\n",
-       "  </tbody>\n",
-       "</table>\n",
-       "<p>6172 rows × 6 columns</p>\n",
-       "</div>"
+      "cell_type": "markdown",
+      "source": [
+        "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/Trusted-AI/AIF360/blob/master/examples/demo_lime.ipynb)"
```
Comment (hoffmansc) [inline examples/sklearn/demo_mdss_bias_scan.ipynb:6]: same here: change to `examples/sklearn/demo_mdss_bias_scan.ipynb`

File: examples/sklearn/demo_mdss_classifier_metric_sklearn.ipynb [hunk 3]
```diff
@@ -1,670 +1,764 @@
...
-       "      <td>F</td>\n",
-       "    </tr>\n",
-       "  </tbody>\n",
-       "</table>\n",
-       "</div>"
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "source": [
+        "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/Trusted-AI/AIF360/blob/master/examples/demo_lime.ipynb)"
```
Comment (hoffmansc) [inline examples/sklearn/demo_mdss_classifier_metric_sklearn.ipynb:6]: same here: change to `examples/sklearn/demo_mdss_classifier_metric_sklearn.ipynb`

File: examples/sklearn/demo_learning_fair_representations.ipynb [hunk 4]
```diff
@@ -1,2025 +1,2140 @@
...
-       "      <td>0.015033</td>\n",
-       "      <td>14</td>\n",
-       "    </tr>\n",
-       "  </tbody>\n",
-       "</table>\n",
-       "</div>"
+      "cell_type": "code",
+      "source": [
+        "# Install AIF360\n",
+        "!pip install 'aif360[LawSchoolGPA]'"
```
Comment (hoffmansc) [inline examples/sklearn/demo_learning_fair_representations.ipynb:27]: we don't need to install LawSchoolGPA here

File: examples/sklearn/demo_mdss_bias_scan.ipynb [hunk 5]
```diff
@@ -1,805 +1,895 @@
...
-   "cell_type": "code",
-   "execution_count": 3,
-   "metadata": {},
-   "outputs": [
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# Install AIF360\n",
+        "!pip install 'aif360[LawSchoolGPA]'"
```
Comment (hoffmansc) [inline examples/sklearn/demo_mdss_bias_scan.ipynb:29]: same here: we don't need LawSchoolGPA

File: examples/sklearn/demo_mdss_classifier_metric_sklearn.ipynb [hunk 6]
```diff
@@ -1,670 +1,764 @@
...
-    {
-     "data": {
-      "text/plain": [
-       "array(['Recidivated', 'Survived'], dtype=object)"
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# Install AIF360\n",
+        "!pip install 'aif360[LawSchoolGPA]'"
```
Comment (hoffmansc) [inline examples/sklearn/demo_mdss_classifier_metric_sklearn.ipynb:51]: same here: we don't need LawSchoolGPA

**Review by hoffmansc — CHANGES_REQUESTED** [review]

File: examples/sklearn/demo_mdss_classifier_metric_sklearn.ipynb (hunk 6 above)
Comment (dharmod) [reply examples/sklearn/demo_mdss_classifier_metric_sklearn.ipynb:51]: I got the message below, so added LAwSchoolGPA
WARNING:root:No module named 'tempeh': fetch_lawschool_gpa will be unavailable. To install, run:
pip install 'aif360[LawSchoolGPA]'

The warning was resolved after adding aif360[LawSchoolGPA]

File: examples/sklearn/demo_mdss_classifier_metric_sklearn.ipynb (hunk 3 above)
Comment (dharmod) [reply examples/sklearn/demo_mdss_classifier_metric_sklearn.ipynb]: Done

File: examples/sklearn/demo_learning_fair_representations.ipynb (hunk 1 above)
Comment (dharmod) [reply examples/sklearn/demo_learning_fair_representations.ipynb]: Done

**Comment by anupamamurthi** [comment]
can you also update the title of the PR so that it is a bit clear

File: examples/sklearn/demo_mdss_classifier_metric_sklearn.ipynb (hunk 6 above)
Comment (hoffmansc) [reply examples/sklearn/demo_mdss_classifier_metric_sklearn.ipynb:51]: It's just a warning but it's not relevant to this code so we can ignore it.

Comment (dharmod) [reply examples/sklearn/demo_mdss_classifier_metric_sklearn.ipynb]: Changes updated

**Pull request merged**